isthmus/src/main/java/io/substrait/isthmus/calcite/SubstraitOperatorTable.java

```diff
@@ -0,0 +1,73 @@
+package io.substrait.isthmus.calcite;
+
+import io.substrait.isthmus.AggregateFunctions;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.apache.calcite.sql.SqlFunctionCategory;
+import org.apache.calcite.sql.SqlIdentifier;
+import org.apache.calcite.sql.SqlKind;
+import org.apache.calcite.sql.SqlOperator;
+import org.apache.calcite.sql.SqlOperatorTable;
+import org.apache.calcite.sql.SqlSyntax;
+import org.apache.calcite.sql.fun.SqlStdOperatorTable;
+import org.apache.calcite.sql.util.SqlOperatorTables;
+import org.apache.calcite.sql.validate.SqlNameMatcher;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+/**
+ * Overrides SQL operator lookups to return Substrait specific functions variants (e.g. {@link
+ * AggregateFunctions#MAX}} when they are available.
+ */
+public class SubstraitOperatorTable implements SqlOperatorTable {
+
+  public static SubstraitOperatorTable INSTANCE = new SubstraitOperatorTable();
```

Comment:
minor: consider giving this class a private constructor if it's not meant to be instantiated by other classes.